To: vim_dev@googlegroups.com
Subject: Patch 7.3.078
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.078
Problem:    Warning for unused variable.
Solution:   Adjuste #ifdefs.
Files:	    src/ops.c


*** ../vim-7.3.077/src/ops.c	2010-12-02 21:43:10.000000000 +0100
--- src/ops.c	2010-12-08 14:21:48.000000000 +0100
***************
*** 2844,2850 ****
--- 2844,2852 ----
      char_u		*p;
      char_u		*pnew;
      struct block_def	bd;
+ #if defined(FEAT_CLIPBOARD) && defined(FEAT_X11)
      int			did_star = FALSE;
+ #endif
  
  				    /* check for read-only register */
      if (oap->regname != 0 && !valid_yank_reg(oap->regname, TRUE))
***************
*** 3127,3133 ****
--- 3129,3137 ----
  
  	clip_own_selection(&clip_star);
  	clip_gen_set_selection(&clip_star);
+ # ifdef FEAT_X11
  	did_star = TRUE;
+ # endif
      }
  
  # ifdef FEAT_X11
*** ../vim-7.3.077/src/version.c	2010-12-08 13:16:58.000000000 +0100
--- src/version.c	2010-12-08 14:22:42.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     78,
  /**/

-- 
A)bort, R)etry, D)o it right this time

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
